cmd/zetcd/zetcd.go

```diff
@@ -73,6 +101,9 @@ func newOracle(etcdAddr, bridgeAddr, oracle string) (p personality) {
 
 func main() {
 	etcdAddr := flag.String("endpoint", "", "etcd3 client address")
+	etcdCertFile := flag.String("certfile", "", "etcd3 cert file")
+	etcdKeyFile := flag.String("keyfile", "", "etcd3 key file")
+	etcdCaFile := flag.String("cafile", "", "etcd3 ca file")
```

Comment:
Actually, never mind. The ones I suggested are v2-style. This conforms to v3's etcdctl